sklearn/utils/testing.py

```diff
@@ -688,6 +692,29 @@ def func(*args, **kwargs):
     return decorator
 
 
+def if_safe_multiprocessing_with_blas(func):
```

Comment:
This is much more conservative now, right? So the CI will not test the parallelization on older numpy / scipy. That is not great, but I guess the benefit is that users will have more robust testing?
But the users might actually run into this problem, so I'm not sure skipping tests is the right thing to do.